docs/views/how-tos/updating-the-kit.html

```diff
@@ -0,0 +1,69 @@
+{% extends 'layouts/layout.html' %}
+
+{% block pageTitle %}
+  Updating the kit - NHS.UK prototype kit
+{% endblock %}
+
+{% block beforeContent %}
+  {% include "how-tos/includes/breadcrumb.html" %}
+{% endblock %}
+
+{% block content %}
+  <div class="nhsuk-grid-row">
+
+    <div class="nhsuk-grid-column-two-thirds">
+
+      <h1>
+        Updating the kit
+      </h1>
+
+      <div class="nhsuk-inset-text">
+        <p>
+          If you have made any changes outside the <strong>app</strong> folder, this process will destroy those changes.
+          We will try and improve the update process to avoid this, but in the meantime you will need to make a note of your changes outside <strong>app</strong>, and add them back after updating.
+        </p>
+      </div>
+
+      <ol>
+        <li>
+          <a href="/docs/install/download-zip">Download the latest prototype kit</a>
```

Comment:
Does this need to say "Download the latest version of the prototype kit"?

Will users get confused thinking there's a new kit altogether?

Reply:
Updated to say "Download the latest version of the prototype kit"